blocks/library/paragraph/index.js

```diff
@@ -252,6 +256,28 @@ export const settings = {
 		],
 	},
 
+	deprecated: [
+		{
+			supports,
+			attributes: {
+				...schema,
+				content: {
+					type: 'string',
+					source: 'html',
+				},
+			},
+			save( { attributes } ) {
+				return attributes.content;
```

Comment:
Interesting, the issue I see about this is that anything (any HTML) could be considered a valid paragraph block

Reply:
>Interesting, the issue I see about this is that anything (any HTML) could be considered a valid paragraph block

Wouldn't it only be tried for the paragraph blocks, since it happens during the parse step for the individual block?

Reply:
Yes, that's true but if I tweak the paragraph block with any markup (say in a code editor, I add an image), this markup will be upgraded and shown in the editor. I don't know if we want to fix this or consider it a rare edge case

Reply:
>Yes, that's true but if I tweak the paragraph block with any markup (say in a code editor, I add an image), this markup will be upgraded and shown in the editor. I don't know if we want to fix this or consider it a rare edge case

Hmm, that's a good point. If we cared to, we could filter the HTML allowed here. Is this something the raw handler would be well-suited for, @iseulde ?

Reply:
Could an example be given here? Not sure I follow entirely. A few weeks ago I was thinking to add the `valid_elements` TinyMCE filter to only include the formatting we allow and if multiline the tag + formatting. This could then be enforced on setting initial content. This whitelist could also be shared with raw handling.

Reply:
I think this is ok, you currently can write plenty of HTML within the paragraph content if you want so, and that is not really different.

Reply:
mmm that's true, I guess we need to fix this separately if we ever want to then. (Maybe apply a filter in the `save` function which would invalidate the block)

Reply:
Yes, this seems like another layer of refinement that we might want to explore separately. But not even sure we should care too much about it.